ramls/items-in-transit.json

```diff
@@ -210,6 +210,15 @@
                 }
               }
             }
+          },
+          "copyNumber": {
```

Comment:
This property is deprecated. The `effectiveCallNumberComponents` already has `copyNumber` property.

Reply:
Just ignore. Mixed it up with `callNumber`